test/jobs/mfa_usage_stats_job_test.rb

```diff
@@ -0,0 +1,36 @@
+require "test_helper"
+
+class MfaUsageStatsJobTest < ActiveJob::TestCase
+  include StatsD::Instrument::Assertions
+
+  setup do
+    create(:user, mfa_level: 0) # non-mfa user
```

Comment:
```suggestion
    create(:user, mfa_level: :disabled)
```
could we use the corresponding level instead of the integer value? That way I think the comments can be removed.

Reply:
I'll switch to using the keys, but I think the comments are still necessary to show which of the 4 categories each line is creating.